### ISSUE TITLE
There are some cases where judge result of Test Run is different, when the expected value of test data is a floating point number

### ISSUE THREAD

**Issue by m0ch1m0ch1** [sachaos/atcoder#17]
# Environment
Version: v0.3.4
OS: ubuntu18.04.4 LTS

# Description of this Issue
There are some cases where judge result of Test Run is different, 
when the expected value of test data is a floating point number. 
Below are examples.
- ABC117-A: https://atcoder.jp/contests/abc117/tasks/abc117_a
```
$ acd abc117/abc117_a test
Case 1: WA
Input:
8 3

Expected:
2.6666666667

Actually:
2.6666666666666665

Exit with: 0
Time: 3.3488ms

Case 2: WA
Input:
99 1

Expected:
99.0000000000

Actually:
99

Exit with: 0
Time: 3.1136ms

Case 3: WA
Input:
1 100

Expected:
0.0100000000

Actually:
0.01

Exit with: 0
Time: 2.8194ms
```
- ABC119-B: https://atcoder.jp/contests/abc119/tasks/abc119_b
```
$ acd abc119/abc119_b test
Case 1: WA
Input:
2
10000 JPY
0.10000000 BTC

Expected:
48000.0

Actually:
48000.000000

Exit with: 0
Time: 3.3426ms

Case 2: WA
Input:
3
100000000 JPY
100.00000000 BTC
0.00000001 BTC

Expected:
138000000.0038

Actually:
138000000.003800

Exit with: 0
Time: 3.1238ms
```

**Comment by m0ch1m0ch1** [comment]
This Issue can be avoided by using the --skip-test option.
However, I think it is better to solve this problem in the long run if possible.

**Comment by sachaos** [comment]
@m0ch1m0ch1 I fixed this issue in https://github.com/sachaos/ac-deck/commit/b82c2fc2c0f18c7c8cdd4a51dfb02e793e10a0ea .
It is rough method, but it will work for almost cases.

The patch is included in v0.3.5 .
Please try that.
